tests/scripts/da/autotune_id_tests.sh

```diff
@@ -64,7 +64,7 @@ function autotune_id_tests() {
 	fi
 	
 	# Cleanup autotune
-	autotune_cleanup  | tee -a ${LOG}
+#	autotune_cleanup  | tee -a ${LOG}
```

Comment:
Is this no longer needed, if so, please remove it.

Reply:
It is needed, had commented it while testing have removed the comment.